src/debugtopus.ts

```diff
@@ -76,11 +76,15 @@ export const prepareTestRun = async ({
     packageRootDir = getPackageRootLevel(appDir);
   }
 
-  const tempDir = path.join(packageRootDir, "temp");
```

Comment:
just curious why is temp now a string template

Reply:
oopsie, wanted to add a UUID first, but complicated things, fixed